### PR TITLE
init

### PR DESCRIPTION
**Pull request by askmike**

### CODE REVIEW

File: contracts/playground/greeter.sol [hunk 1]
```diff
@@ -0,0 +1,26 @@
+pragma solidity ^0.4.0;
```
Comment (ale316) [inline contracts/playground/greeter.sol]: should get rid of play files

File: contracts/ico/ico.sol [hunk 2]
```diff
@@ -0,0 +1,11 @@
+pragma solidity ^0.4.0;
```
Comment (ale316) [inline contracts/ico/ico.sol]: how many contracts are we gonna need approx? might not make sense to have a folder for each contract. Doesn't matter too much though

Comment (askmike) [reply contracts/ico/ico.sol]: fixed.

File: contracts/playground/greeter.sol (hunk 1 above)
Comment (askmike) [reply contracts/playground/greeter.sol]: fixed.

**Pull request merged**